src/test/java/de/tum/in/www1/artemis/TextSubmissionIntegrationTest.java

```diff
@@ -378,4 +378,18 @@ private void checkDetailsHidden(TextSubmission submission, boolean isStudent) {
             assertThat(((StudentParticipation) submission.getParticipation()).getStudent()).as("student of participation is hidden").isEmpty();
         }
     }
+
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void testPlagiarismCheck() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968849)